database/models/token_blocklist.py

```diff
@@ -1,12 +1,17 @@
 from .base import Base, Column, Integer, String, ForeignKey, Boolean, DateTime
+from sqlalchemy.orm import relationship
 import datetime 
 
-class Token_Blocklist(Base):
+class TokenBlocklist(Base):
     __tablename__ = 'Token_blocklist'
+
     id = Column(Integer, primary_key=True, autoincrement=True)
     user_id = Column(Integer, ForeignKey("Users.user_id"))
     jti = Column(String, nullable=False, index=True)
     token_type = Column(String(10), nullable=False)
     revoked = Column(Boolean, default=False, nullable=False)
     expires_at = Column(DateTime, nullable=False)
-    updated_at = Column(DateTime)  
\ No newline at end of file
+    updated_at = Column(DateTime)  
+
```

Comment:
For god's sake, stop beating the shit out of your `Enter` button. One empty line is plenty